### PR TITLE
Feature/mem 07 l xml

### PR DESCRIPTION
**Pull request by barricade86**

### CODE REVIEW

File: mem-07-l.xml [hunk 1]
```diff
@@ -271,7 +271,9 @@
       <column name="examples_de"></column>
       <column name="examples_fa"></column>
       <column name="examples_sv"></column>
-      <column name="examples_ru"></column>
+      <column name="examples_ru">
+▶ {beq may' ta vIlajpu'} "Я принял запись экипажа о битве"[2]
```
Comment (dlyongemallo) [inline mem-07-l.xml:275]: Should be preserved:
▶ {ghu' Dachupbogh vIlaj.:sen}

File: mem-07-l.xml [hunk 2]
```diff
@@ -484,7 +486,8 @@
       <column name="examples_de"></column>
       <column name="examples_fa"></column>
       <column name="examples_sv"></column>
-      <column name="examples_ru"></column>
+      <column name="examples_ru">
+▶ {HurDaq tu'HomI'raHvetlh yIlan!:sen:nolink} "Убери отсюда этот хлам!"[2]</column>
```
Comment (dlyongemallo) [inline mem-07-l.xml:490]: Should be preserved:
▶ {nuqDaq Dochvam vIlan?:sen}

File: mem-07-l.xml [hunk 3]
```diff
@@ -809,7 +812,8 @@ In English slang, a "lat"rine is sometimes called a "shrine".</column>
       <column name="examples_de"></column>
       <column name="examples_fa"></column>
       <column name="examples_sv"></column>
-      <column name="examples_ru"></column>
+      <column name="examples_ru">
```
Comment (dlyongemallo) [inline mem-07-l.xml:815]: Should be preserved:
▶ {reH latlh qabDaq qul tuj law' Hoch tuj puS.:sen}[2]

File: mem-07-l.xml [hunk 4]
```diff
@@ -1096,7 +1110,28 @@ The following example shows that the verb of quality can take a suffix:
...
```
Comment (dlyongemallo) [inline mem-07-l.xml:1113]: Missing a section of examples.

File: mem-07-l.xml [hunk 5]
```diff
@@ -2706,7 +2758,7 @@ This word can be used figuratively. There is an idiom, {let mInDu'Daj; Separmey
       <column name="search_tags">be general</column>
       <column name="search_tags_de">allgemein</column>
       <column name="search_tags_fa"></column>
-      <column name="search_tags_sv"></column>
+      <column name="search_tags_sv">быть общим</column>
```
Comment (dlyongemallo) [inline mem-07-l.xml:2761]: Should be `search_tags_ru` not `search_tags_sv`.

File: mem-07-l.xml [hunk 6]
```diff
@@ -5737,12 +5803,12 @@ Some of the verb suffixes have a precise meaning when used with this verb, when
...
       <column name="definition_fa">quadrotriticale [AUTOTRANSLATED]</column>
       <column name="definition_sv">kvadrotrikale</column>
-      <column name="definition_ru"></column>
+      <column name="definition_ru">Квадротритикаль</column>
       <column name="definition_zh_HK">quadrotriticale [AUTOTRANSLATED]</column>
       <column name="synonyms"></column>
       <column name="antonyms"></column>
       <column name="see_also">{tIr:n}</column>
-      <column name="notes">Quadrotriticale is a four-lobed hybridization of wheat (genus Triticum) and rye (genus Secale).</column>
+      <column name="notes">Квадротритикаль это четырехдольная гибридизация пшеницы (рода Triticum) и ржи (род Secale).</column>
```
Comment (dlyongemallo) [inline mem-07-l.xml:5811]: Should be under `notes_ru`.

**Review by dlyongemallo — COMMENTED** [review]
Thanks for doing this! I spotted a few minor errors (which I've fixed).

File: mem-07-l.xml (hunk 4 above)
Comment (barricade86) [reply mem-07-l.xml:1113]: The content of examples_ru is below, or i didn't unterstood your comment

Comment (dlyongemallo) [reply mem-07-l.xml:1113]: This is missing the entire section from:
▶ {batlh potlh law' yIn potlh puS.:sen}
to:
▶ {'IwwIj jeD law' 'IwlIj jeD puS.:sen}

**Pull request merged**